fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -425,8 +428,8 @@ public Response createOrReplaceObjectRdf(
                         links, checksums, requestBodyStream, extContent);
             }
         } else {
-            final Model model = httpRdfService.bodyToInternalModel(externalPath(), requestBodyStream,
-                requestContentType);
+            final Model model = httpRdfService.bodyToInternalModel(externalUri, requestBodyStream,
+                requestContentType, identifierConverter);
```

Comment:
For consistency and safety, probably best to use the `identifierConverter()` method.

Reply:
This is done.